src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 2;
```

Comment:
The name `SHORT_TERM` is misleading because it implies a duration, but the value `2` does not represent a duration that qualifies for a discount. According to the task, the discount starts at 3 days. Consider renaming the constant to reflect the correct threshold for the short-term discount or adjust the value to match the task requirements.